frontend/packages/dev-console/src/components/formik-fields/DropdownField.tsx

```diff
@@ -31,7 +47,9 @@ const DropdownField: React.FC<DropdownFieldProps> = ({ label, helpText, required
           setFieldValue(props.name, value);
           setFieldTouched(props.name, true);
         }}
+        isValid={!isDuplicateEntry}
       />
+      {isDuplicateEntry && <p style={{color:"red", fontStyle: "italic"}}>Duplicate Entry Detected!</p>}
```

Comment:
could you change to "Duplicate Revision detected."
- lowercase d in "detected"
- period, not exclamation point

Reply:
* `DropdownField` is a generic component. Adding Revisions in the error message would make it specific to Knative/Revisions/TrafficSplitting.
* Replaced the exclamation mark with period

Reply:
@nemesis09 could you change from "Duplicate Entry detected." to "Duplication Revision detected." ?

Reply:
@serenamarie125 I don't think it can be changed in the DropdownField component as that would make the component specific to Knative/Revision/TrafficSplitting.
I can pass a prop like `itemHeader` to pass down "Revisions" to rendered in the error message.